UIKitTask/Sources/Models/DataStorageService.swift

```diff
@@ -0,0 +1,148 @@
+// DataStorageService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Тестовые данные для ленты постов и списка новых уведомлений,  заглушки. Имитация сети
+final class DataStorageService {
+    // MARK: Private Properties
```

Comment:
```suggestion

/// Тестовые данные для ленты постов и списка новых уведомлений,  заглушки. Имитация сети
final class DataStorageService {
    // MARK: Private Properties
```
final class StorageService {